lib/cylc/review_dao.py

```diff
@@ -539,7 +558,11 @@ def get_suite_cycles_summary(
             " sum(" + states_stmt["success"] + ") AS n_success,"
             " sum(" + states_stmt["fail"] + ") AS n_fail"
             " FROM task_states" +
-            " GROUP BY cycle")
+            " GROUP BY cycle" +
+            " HAVING sum(" + states_stmt["active"] + ")>0" +
```

Comment:
Replaces the logic at line 556 of the old code. Only retrieve info for cycles with non-waiting tasks.